setup.py

```diff
@@ -18,6 +18,7 @@
 
     install_requirements = [
         "vivarium>=0.10.14",
+        "vivarium_cluster_tools>=1.3.3",
```

Comment:
Should this be updated to the most recent version? Since 1.3.7 gets installed anyway.

Reply:
Shouldn't the version specified get installed?  I'm also not sure if VCT should be in install requirements.

Reply:
I don't think you should update it from a version you've already run with, until you actually run it with the new version.

@albrja VCT is needed in `make_artifacts`.

Reply:
The pins should be "==" pins so we can guarantee what version is used